tests/test_utils.py

```diff
@@ -16,3 +16,14 @@ def test_base64encode_bytes():
 
     assert isinstance(encoded_string, str)
     assert encoded_string == 'dGhpcyBpcyBieXRlcyB0byBlbmNvZGU=\n'
+
+
+@pytest.mark.parametrize("test_input,expected", [
+    ('WEBRip 1080p', 'webrip1080p'),
+    ('WEBRip-1080p', 'webrip1080p'),
+    ('WEBRip_1080p', 'webrip1080p'),
+    ('', ''),
+    (None, ''),
+])
+def test_test_sanitize_quantity(test_input, expected):
```

Comment:
Please make a separate test module for the tracker if you want to have additional test (if integration tests through ``test_urls`` are not sufficient).